powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationListForUserRequest.java

```diff
@@ -34,10 +34,10 @@ public class OperationListForUserRequest {
 
     private String userId;
     private List<String> applications = new ArrayList<>();
-    private String activationFlag;
```

Comment:
🤔 Ok, it was mentioned in the issue that we may remove it right away without deprecation.